puppet/environments/development/manifests/site.pp

```diff
@@ -0,0 +1,13 @@
+node default {
+
+  class { ['nodejs', 'chromium']: }
+
+  janus::role::standalone { 'janus':
+    hostname => 'janus-gateway-js.dev.cargomedia.ch',
+  }
+
+  package { ['gulp', 'testem']:
```

Comment:
Oh I just meant that here we install `gulp` globally, with latest (or whatever defined in puppet package) version. I assume it's safer to install nodejs deps for project using `package.json` and run them locally - `node_modules/.bin/gulp`. This way you sure your gulpfile is in the same version as gulp binary.

Same goes for `browserify` in `package.json`. We use global binary (?), while we could/should use local one.

Please correct me if I'm wrong.